packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -81,7 +81,7 @@ export default class Meetings extends SparkPlugin {
       // };
 
       // rather then locus object change to locus url
-      this.create(data.locus, LOCUS_ID).then(() => {
+      this.create(data.locus, LOCUS_ID).then((meeting) => {
         if (data.eventType === LOCUSEVENT.DIFFERENCE) {
           // its a delta object and we have a new meeting
           meeting.locusInfo.initialSetup(data.locus, meeting);
```

Comment:
meeting is already defined above. We should either always pass meeting down through the chain of promises, or not use the arg function here. I think we should do the first one (always return meeting from each step in the promise chain) since it can be undefined.

Reply:
I think i need to change the name to newMeeting . We create a meeting object if there is no meeting object present . The names are little confusing